docs/authoring.md

```diff
@@ -211,17 +211,17 @@ The `generator` property from `package.json` file must contain a JSON object tha
     }
   },
   "conditionalFiles": {
-    "src/api/adapters/amqp.js": {
+    "template/dir/relative/path/to/file/test-amqp.js": {
       "subject": "server.protocol",
       "validation": {
         "const": "amqp"
       }
     },
-    "src/api/adapters/mqtt.js": {
-      "subject": "server.protocol",
-      "validation": {
-        "const": "mqtt"
-      }
+    "template/dir/relative/path/to/file/support.html": {
```

Comment:
Without `template` in the beginning, because of https://github.com/asyncapi/generator/blob/736bfa0108d158da07d86acaffbc57438c402f73/lib/generator.js#L546

Reply:
ok, I think I made it a bit too confusing, I meant here in example that path must be relative to template dir. I'll fix